src/key/key.js

```diff
@@ -881,7 +881,7 @@ class Key {
       throw new Error("Subkey could not be encrypted here, please encrypt whole key");
     }
     if (util.getWebCryptoAll() && options.rsaBits < 2048) {
-      throw new Error('When using webCrypto rsaBits should be 2048 or 4096, found: ' + options.rsaBits);
+      throw new Error(`rsaBits should be at least 2048, got: ${options.rsaBits}`);
```

Comment:
Either put a RSA size lower bound regardless of webcrypto (config?), or skip this check altogether

Reply:
Yeah, we could make a config option for the minimum rsa bits indeed :+1: And then lower it in the tests. And I think we could also remove the check for web crypto in the tests, as web crypto also supports rsa bits < 2048 (nowadays?). Though I think it doesn't necessarily have to happen in this PR